FiiPrezent/ViewModels/ErrorViewModel.cs

```diff
@@ -1,11 +1,9 @@
-using System;
-
-namespace FiiPrezent.Models
-{
-    public class ErrorViewModel
-    {
-        public string RequestId { get; set; }
-
-        public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);
-    }
-}
+namespace FiiPrezent.ViewModels
```

Comment:
I'm not sure about the moving these to the "ViewModels" folder. The convention in MVC is that the (view) models stay in the "Model" folder :)

Reply:
A model is a class representation and has no knowledge of the View (like Event.cs) and a ViewModel is a data binder between the view and the model (EventViewModel.cs). Also, the viewModels contains attributes like DisplayName or Required. This is why I like to separate them. I think this is the MVVM pattern.

Reply:
MVVM is used in frameworks that allow two way binding (Model->View and View->Model hence the name) like WPF. MVC doesn't work that way.
In our case we use different models for GET and POST. The decision to name them ViewModels was mine to emphasise that the service layer should not know about them. This is definitely not MVVM.